docs/self-hosted-appcircle/configure-server/_firewalld-configuration.mdx

```diff
@@ -0,0 +1,23 @@
+If you are using `Firewalld`, you need to open the ports below according to your server configuration.
+
+- If you plan to run the Appcircle server with HTTPS:
+
+```bash
+sudo firewall-cmd --add-port=80/tcp --permanent
+sudo firewall-cmd --add-port=443/tcp --permanent
+sudo firewall-cmd --reload
+```
+
+- If you plan to run the Appcircle server with HTTP:
+
+```bash
+sudo firewall-cmd --add-port=80/tcp --permanent
+sudo firewall-cmd --add-port=6379/tcp --permanent
```

Comment:
Is that right to open port `6379` on a server that can be accessed from internet? (DMZ)
- https://deploy-preview-524--reverent-galileo-8ef035.netlify.app/self-hosted-appcircle/configure-server/advanced-configuration/store-dist-dmz#firewall-configuration

<img width="843" alt="Screenshot 2024-07-10 at 12 37 44" src="https://github.com/appcircleio/appcircle-docusaurus/assets/7355362/4b2e65db-bb1a-48ff-b78d-50127a4170e9">

Reply:
:ok: